enterprise/cmd/symbols/main.go

```diff
@@ -1,160 +1,11 @@
 package main
 
 import (
-	"context"
-	"database/sql"
-	"net/http"
-	"strings"
-
-	"github.com/sourcegraph/go-ctags"
-	"github.com/sourcegraph/log"
-
-	"github.com/sourcegraph/sourcegraph/cmd/symbols/fetcher"
-	symbolsGitserver "github.com/sourcegraph/sourcegraph/cmd/symbols/gitserver"
-	symbolsParser "github.com/sourcegraph/sourcegraph/cmd/symbols/parser"
 	"github.com/sourcegraph/sourcegraph/cmd/symbols/shared"
-	"github.com/sourcegraph/sourcegraph/cmd/symbols/types"
-	"github.com/sourcegraph/sourcegraph/enterprise/internal/rockskip"
-	"github.com/sourcegraph/sourcegraph/internal/conf"
-	"github.com/sourcegraph/sourcegraph/internal/conf/conftypes"
-	"github.com/sourcegraph/sourcegraph/internal/database"
-	connections "github.com/sourcegraph/sourcegraph/internal/database/connections/live"
-	"github.com/sourcegraph/sourcegraph/internal/env"
-	"github.com/sourcegraph/sourcegraph/internal/goroutine"
-	"github.com/sourcegraph/sourcegraph/internal/observation"
-	"github.com/sourcegraph/sourcegraph/internal/search"
-	"github.com/sourcegraph/sourcegraph/internal/search/result"
+	shared_symbols "github.com/sourcegraph/sourcegraph/cmd/symbols/shared"
+	shared_symbols_enterprise "github.com/sourcegraph/sourcegraph/enterprise/cmd/symbols/shared"
```

Comment:
Nit: I don't think we [underscores in imports](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+lang:go+%5Cw%2B%5Cs%22github%5C.com%5C/sourcegraph%5C/sourcegraph%5C/enterprise.*%22%5Cn&patternType=regexp&sm=1&groupBy=path), even at the cost of readability. So for sake of consistency I'd go with [that](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+lang:go+%5Cw%2B%5Cs%22github%5C.com%5C/sourcegraph%5C/sourcegraph%5C/enterprise.*shared%22%5Cn&patternType=regexp&sm=1&groupBy=path) here too:

```suggestion
	symbolsshared "github.com/sourcegraph/sourcegraph/cmd/symbols/shared"
	entsymbolsshared "github.com/sourcegraph/sourcegraph/enterprise/cmd/symbols/shared"
```

Reply:
argh. auto-merge & approve.